deploy/olm-catalog/ibm-commonui-operator/1.3.0/ibm-commonui-operator.v1.3.0.clusterserviceversion.yaml

```diff
@@ -191,6 +206,41 @@ spec:
         - description: Displays names of pods associated with the Legacy Header service
           displayName: Pod Names
           path: nodes
+    - description: CloudPakSwitcher is the Schema for the cloupakswithcer API
+      kind: CloudPakSwitcher
+      name: cloudpakswitchers.operators.ibm.com
+      version: v1alpha1
+      displayName: CloudPakSwitcher service
+      resources:
+      - kind: Service
```

Comment:
this part of the list should only display resources the new api will create. It won't create any configmaps, ingresses, or commonwebui resources correct?